src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -42,6 +43,19 @@ public ConfigurationFeatureDefinitionProvider(IConfiguration configuration, ILog
                 () => _stale = 1);
         }
 
+        /// <summary>
+        /// The option that controls the behavior when "FeatureManagement" section in the configuration is missing.
+        /// </summary>
+        public bool UseTopLevelConfiguration
+        {
+            get => _useTopLevelConfiguration;
+
+            init
+            {
+                _useTopLevelConfiguration = value;
```

Comment:
This is unnecessary since no validation is performed. We should be able to just have


```
public bool UseTopLevelConfiguration { get; init; }
```